pkgs/games/hmcl/default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
```

Comment:
use `makeBinaryWrapper` in place of `makeWrapper` for a better compatibility with Darwin platforms.

Reply:
OK, I used it